modules/gitops-release-notes-1-19-0.adoc

```diff
@@ -0,0 +1,158 @@
++
+link:https://issues.redhat.com/browse/GITOPS-7704[GITOPS-7704]
+
+Argo CD Image Updater available as a Technology Preview::
+With this update, Argo CD Image Updater is now available as a Technology Preview feature in {gitops-title}. Argo CD Image Updater automatically updates container images for Kubernetes workloads managed by Argo CD. It tracks image versions in applications as defined by ImageUpdater CRs and updates it by applying parameter overrides through the Argo CD API or by using the Git write-back method. The image updater supports applications built with Kustomize or Helm. In this release, the {gitops-title} Operator provides a productized container image of the Argo CD Image Updater and introduces a new property in the Argo CD CR to enable or disable the feature.
++
+The following example shows how to configure the Argo CD Image Updater feature in the Argo CD CR:
++
+[source,YAML]
+----
+spec:
+  sso:
+    provider: dex
+    dex:
+      volumeMounts:
+        - name: custom-cert
+          mountPath: /etc/dex/ssl
+          readOnly: true
+      volumes:
+        - name: custom-cert
+          secret:
+            secretName: dex-tls-secret
+----
++
+For more details, see the Overview of the Argo CD Image Updater in the upstream documentation.
++
+link:https://issues.redhat.com/browse/GITOPS-7491[GITOPS-7491]
+
+Configurable resource limits and requests for GitOps plugin components::
+With this update, you can configure resource requests and limits for the cluster and {gitops-shortname} plugin components through the `GitOpsService` CR. Administrators can define these values by using the following fields:
+* `spec.consolePlugin.backend.resources`
+* `spec.consolePlugin.gitopsPlugin.resources`
++
+This enhancement allows administrators to assign distinct or identical resource allocations to the {gitops-shortname} plugin components, providing greater flexibility in resource management.
++
+The following example shows how to configure resource requests and limits for the console plugin backend and GitOps plugin components in the `GitopsService` CR:
++
+[source,yaml]
+----
+apiVersion: pipelines.openshift.io/v1alpha1
+kind: GitopsService
+metadata:
+  name: cluster
+spec:
+  consolePlugin:
+    backend:
+      resources:
+        limits:
+          cpu: 100m
+          memory: 1Gi
+        requests:
+          cpu: 100m
+          memory: 1Gi
+    gitopsPlugin:
+      resources:
+        limits:
+          cpu: 200m
+          memory: 2Gi
+        requests:
+          cpu: 100m
+          memory: 1Gi
+----
++
+link:https://issues.redhat.com/browse/GITOPS-7416[GITOPS-7416]
```

Comment:
Same as previous comment. Can we reference the main documentation page?

Reply:
Yes, answered in this [comment](https://github.com/openshift/openshift-docs/pull/103017#discussion_r2567256174).

Reply:
Fixed, updated the Additional Resources section with the link.